src/core/simplify/effectivearea.h

```diff
@@ -68,7 +68,7 @@ struct MINHEAP
 };
 
 /**
- * Structure to hold pointarray and it's arealist.
+ * Structure to hold pointarray and its arealist.
```

Comment:
Make that a `point array` as well, I guess.

Reply:
I thought it was a variable but seems not. Should `arealist` also become `area list` (not sure what it's in the code)?